src/Core/src/Handlers/SwipeItemMenuItem/SwipeItemMenuItemHandler.iOS.cs

```diff
@@ -1,34 +1,74 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Text;
 using CoreGraphics;
 using Microsoft.Extensions.Logging;
-using ObjCRuntime;
 using UIKit;
 
 namespace Microsoft.Maui.Handlers
 {
+	// TODO: Make this public on .NET 8
```

Comment:
We're merging this for net8, right? So this can be public?

Reply:
Yeah, I think what we have been doing is main is public, and then the backports are edited to be internal

Reply:
made it public!